src/components/ConnectExchangeView/ExchangeAccountConnect/ExchangeAccountConnect.js

```diff
@@ -115,6 +116,11 @@ const ExchangeAccountConnect = () => {
       .exchangeAdd(payload)
       .then(() => {
         setTempMessage(<FormattedMessage id={"accounts.connected.success"} />);
+        const exchangePayload = {
+          token: storeSession.tradeApi.accessToken,
+        };
+        dispatch(getUserExchanges(exchangePayload));
+        dispatch(getUserData(exchangePayload));
```

Comment:
Any reason we need to dispatch `getUserData` there?

Reply:
@cwagner22  there was a task to update user data whenever user connected/removed an exchange. there are some properties in user data that are updated according to that.